mandatory/3-function-output.js

```diff
@@ -1,16 +1,19 @@
 // Add comments to explain what this function does. You're meant to use Google!
 function getNumber() {
-  return Math.random() * 10;
+  return Math.random() * 10;      
 }
+// The function above will return a random integer from 0 and 9, The random() method returns a random number from 0 (inclusive) up to but not including 1 (exclusive)
 
 // Add comments to explain what this function does. You're meant to use Google!
 function s(w1, w2) {
   return w1.concat(w2);
 }
+// The concat() method above is used to join two or more strings, so the function will return a new string of w1 + w2.
```

Comment:
The function `s` only has 2 arguments, so you can't join more than 2 strings.

Reply:
@louisechow you are right, I should be careful with how I write my answer, in that particual example it will only be used to concatenate the two parameters w1 and w2.